pkg/cmd/registry/rule/list/list.go

```diff
@@ -0,0 +1,181 @@
+	Status:      "DISABLED",
+}
+
+// NewListCommand creates a new command to view status of rules
+func NewListCommand(f *factory.Factory) *cobra.Command {
+
+	opts := &options{
+		IO:         f.IOStreams,
+		Config:     f.Config,
+		Connection: f.Connection,
+		Logger:     f.Logger,
+		localizer:  f.Localizer,
+		Context:    f.Context,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Short:   f.Localizer.MustLocalize("registry.rule.list.cmd.description.short"),
+		Long:    f.Localizer.MustLocalize("registry.rule.list.cmd.description.long"),
+		Example: f.Localizer.MustLocalize("registry.rule.list.cmd.example"),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, _ []string) (err error) {
+
+			cfg, err := opts.Config.Load()
+			if err != nil {
+				return err
+			}
+
+			instanceID, ok := cfg.GetServiceRegistryIdOk()
+			if !ok {
+				return opts.localizer.MustLocalizeError("artifact.cmd.common.error.noServiceRegistrySelected")
+			}
+
+			opts.registryID = instanceID
+
+			return runList(opts)
+		},
+	}
+
+	flags := rulecmdutil.NewFlagSet(cmd, f)
+
+	flags.AddRegistryInstance(&opts.registryID)
+
+	flags.AddArtifactID(&opts.artifactID)
+	flags.AddGroup(&opts.group)
+
+	return cmd
+
+}
+
+func runList(opts *options) error {
+
+	var httpRes *http.Response
+	var newErr error
+	var enabledRules []registryinstanceclient.RuleType
+
+	var ruleErr = &rulecmdutil.RegistryRuleError{
+		Localizer:  opts.localizer,
+		InstanceID: opts.registryID,
+	}
+
+	conn, err := opts.Connection(connection.DefaultConfigRequireMasAuth)
+	if err != nil {
+		return err
+	}
+
+	dataAPI, _, err := conn.API().ServiceRegistryInstance(opts.registryID)
+	if err != nil {
+		return err
+	}
+
+	if opts.artifactID == "" {
+
+		opts.Logger.Info(opts.localizer.MustLocalize("registry.rule.list.log.info.fetching.globalRules"))
+
+		req := dataAPI.AdminApi.ListGlobalRules(opts.Context)
+
+		enabledRules, httpRes, newErr = req.Execute()
+		if httpRes != nil {
+			defer httpRes.Body.Close()
+		}
+	} else {
+
+		opts.Logger.Info(opts.localizer.MustLocalize("registry.rule.list.log.info.fetching.artifactRules"))
+
+		req := dataAPI.ArtifactRulesApi.ListArtifactRules(opts.Context, opts.group, opts.artifactID)
+
+		enabledRules, httpRes, newErr = req.Execute()
+		if httpRes != nil {
+			defer httpRes.Body.Close()
+		}
+	}
+
+	if newErr != nil {
+		if httpRes == nil {
+			return registrycmdutil.TransformInstanceError(newErr)
+		}
+
+		switch httpRes.StatusCode {
+		case http.StatusNotFound:
+			return ruleErr.ArtifactNotFoundError(opts.artifactID)
+		default:
+			return registrycmdutil.TransformInstanceError(newErr)
+		}
+	}
+
+	for _, rule := range enabledRules {
+		if rule == ruleValidity {
+			validityRuleStatus.Status = "ENABLED"
+		}
+		if rule == ruleCompatibility {
+			compatibilityRuleStatus.Status = "ENABLED"
+		}
+	}
+
+	var adminRules = []ruleRow{validityRuleStatus, compatibilityRuleStatus}
+
+	opts.Logger.Info()
+	dump.Table(opts.IO.Out, adminRules)
```

Comment:
The other tables are built from a response which has array of objects. The rule list just returns an array of enabled rule types ( response: `["VALIDITY", "COMPATIBILITY" ]`). The table is constructed to have something similar to what the UI has.
I think having yaml and json for it will not add much value.